model/task_lifecycle.go

```diff
@@ -999,6 +1001,36 @@ func logTaskEndStats(ctx context.Context, t *task.Task) error {
 	return nil
 }
 
+// getVersionCtxForTracing returns a context with version attributes for tracing
+func getVersionCtxForTracing(ctx context.Context, v *Version, project string) (context.Context, error) {
+	if v == nil {
+		return nil, errors.New("version is nil")
+	}
+
+	timeTaken, makespan, err := v.GetTimeSpent()
+	if err != nil {
+		return nil, errors.Wrap(err, "getting time spent")
+	}
+	timeTakenMs := int64(timeTaken.Round(time.Second) / time.Millisecond)
```

Comment:
Would we consider using seconds instead of milliseconds? I feel like there's no world where a version is fewer than one second long, and this would improve readability here. Then we can also use the .Second() function, and use the Int attribute as opposed to Int64

Reply:
It's true, we probably don't need this level of granularity. I doubt any performance improvement will consider 1 m 5 seconds versus 1 m 5.5 seconds. 
I switched it over to seconds.